src/Selenium2Library/keywords/alert.py

```diff
@@ -1,21 +1,23 @@
 import time
+
 from selenium.common.exceptions import WebDriverException
 from selenium.webdriver.support.ui import WebDriverWait
 from selenium.webdriver.support import expected_conditions as EC
 
-from .keywordgroup import KeywordGroup
+from Selenium2Library.base import Base
+from Selenium2Library.robotlibcore import keyword
```

Comment:
Already commented once, but I think `from .module import something` is better than `from Selenium2Library.module import something`. The former has less noise and require no changes if/when we change the library name.